WalletWasabi.Fluent/ViewModels/Dialogs/ManualCoinJoinProfileDialogViewModel.cs

```diff
@@ -22,7 +22,7 @@ public ManualCoinJoinProfileDialogViewModel(CoinJoinProfileViewModelBase current
 
 		_timeFrames = new[]
 		{
-			new TimeFrameItem("Hours", TimeSpan.Zero),
+			new TimeFrameItem("Fastest", TimeSpan.Zero),
```

Comment:
```suggestion
			new TimeFrameItem("Always", TimeSpan.Zero),
```

Isn't that more accurate then?

Reply:
makes sense, that's better